### PR TITLE
mixed-cpu-node-plugin: initial prow ci setup

### PR DESCRIPTION
**Pull request by Tal-or**
New config files for setup prow ci
for mixed-cpu-node-plugin repo under openshift-kni org

### CODE REVIEW

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-kni-mixed-cpu-node-plugin-main-ci-e2e

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-kni-mixed-cpu-node-plugin-main-images

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-kni-mixed-cpu-node-plugin-main-ci-e2e

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-kni-mixed-cpu-node-plugin-main-ci-e2e

File: ci-operator/config/openshift-kni/mixed-cpu-node-plugin/openshift-kni-mixed-cpu-node-plugin-main.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+base_images:
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make fmt vet build-plugin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
```
Comment (ffromani) [inline ci-operator/config/openshift-kni/mixed-cpu-node-plugin/openshift-kni-mixed-cpu-node-plugin-main.yaml:9]: we should probably stop using this system, inherited ages ago from earliest PAO and never really re-evaluated. Not blocking for this PR though.

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**